tzlocal/__init__.py

```diff
@@ -1,7 +1,34 @@
 import sys
-if sys.platform == 'win32':
-    from tzlocal.win32 import get_localzone, reload_localzone
-elif 'darwin' in sys.platform:
-    from tzlocal.darwin import get_localzone, reload_localzone
-else:
-    from tzlocal.unix import get_localzone, reload_localzone
+
+__version__ = "1.5dev0"
+
+# Cache variable
+_cache_tz = None
+
+def get_localzone():
+    """Returns the zoneinfo-based tzinfo object that matches the local system's timezone."""
+    global _cache_tz
+
+    if _cache_tz is not None:
+        return _cache_tz
+
+    # Avoid importing sub-modules at load time, breaks running setup.py
```

Comment:
Only if you import the package from setup.py, which you shouldn't do.

Reply:
As said, importing `__version__` in `setup.py` is just a small DRY thing.

There are reasons why you shouldn't import stuff in `__init__`, in general I keep them as empty as possible. In this case, it's a matter of keeping an API.

http://foobar.lu/wp/2012/05/13/a-comprehensive-step-through-python-packaging-a-k-a-setup-scripts/